bindings/python/spatial/explog.hpp

```diff
@@ -23,9 +23,10 @@ namespace pinocchio
     }
     
     template<typename Matrix3Like>
-    typename EIGEN_PLAIN_TYPE(Matrix3Like) Jlog3_proxy(const Eigen::MatrixBase<Matrix3Like> & M)
+    typename Eigen::Matrix<typename EIGEN_PLAIN_TYPE(Matrix3Like)::Scalar,3,3,EIGEN_PLAIN_TYPE(Matrix3Like)::Options>    
```

Comment:
I don't know why you have to change here?

Reply:
I had to do this in order to be able to use `MatrixXd` as template parameter instead of `Matrix3d`, which I did for the reasons explained above